portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -38,15 +49,49 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String email = userService.getCurrentUser().getEmail();
     String title = request.getParameter("title");
     long timestamp = System.currentTimeMillis();
+    // Get the URL of the image that the user uploaded to Blobstore.
+    String imageUrl = getUploadedFileUrl(request, "image");
```

Comment:
What is the "image" magic constant from?

Reply:
Image is the name assigned to the value by the input button.